packages/gamut/src/Video/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useRef } from 'react';
+import ReactPlayer from 'react-player';
+import PlayIcon from '../Icon/icons/PlayIcon';
+import cx from 'classnames';
+import styles from './styles/index.module.scss';
+
+/** render the play button with an overlay */
+const OverlayPlayButton = () => {
+  return (
+    <div className={styles.overlay}>
+      <PlayIcon className={styles.hoverButton} />
+    </div>
+  );
+};
+
+type ContentfulAsset = {
+  id: string;
+  content_type?: string;
+  title: string;
+  url: string;
+};
+export type VideoProps = {
+  videoUrl: string;
+  videoTitle: string;
+  placeholderImage?: ContentfulAsset;
+  autoplay?: boolean;
+  controls?: boolean;
+  loop?: boolean;
+  muted?: boolean;
+  componentStyle?: string;
```

Comment:
Per our normal prop names:

```suggestion
  className?: string;
```